release/workspace_snippet.tmpl

```diff
@@ -0,0 +1,19 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+${http_archive_statement}
+
+load("@buildifier_prebuilt//:deps.bzl", "buildifier_prebuilt_deps")
+
+buildifier_prebuilt_deps()
+
+load("@bazel_skylib//:workspace.bzl", "bazel_skylib_workspace")
+
+bazel_skylib_workspace()
+
+load("@cgrindel_bazel_starlib//:deps.bzl", "bazel_starlib_dependencies")
```

Comment:
for some reason I was thinking this was actually only a developer dependency? i see it's in deps.bzl as well, but I don't see how users would hit it, what am I missing?

Reply:
You are correct. I mistakenly added it because the dep showed up outside of a test target. However, as you rightly pointed out, the only usage is in the release targets and test targets. 

I confirmed that the simple example does not load cgrindel/bazel-starlib and all tests are green. So, I removed it from the workspace snippet template. Sorry about that.